crates/phactory/src/system/master_key.rs

```diff
@@ -59,22 +75,30 @@ pub fn try_unseal(
         Some(data) => data,
         None => {
             warn!("No sealed master key");
-            return None;
+            return vec![];
         }
     };
```

Comment:
Should the key handover also deal with the GK master key? or encrypt the master key with the worker identity key(maybe derived) rather than the mrenclave protectedfs, or otherwise, GKs would have to resync twice at each pruntime upgrade.

Reply:
Add a TODO.